aiohttp/web_request.py

```diff
@@ -613,7 +613,10 @@ def body_exists(self) -> bool:
                 field_ct = field.headers.get(hdrs.CONTENT_TYPE)
```

Comment:
It looks like according to https://tools.ietf.org/html/rfc7578#section-4.4 it should be
```suggestion
                 field_ct = field.headers.get(hdrs.CONTENT_TYPE, 'plain/text')
```
shouldn't it?

Reply:
It's the default according to the RFC, but only matters to whoever processes the contents of the file. I mentioned this in #4089: Setting the content type means the user will not be able to distinguish between a file uploaded with the content type of `text/plain` and one being uploaded with no content type. I'm not sure what the value would be in setting it, especially since it might be wrong.

What users typically do is use the file name / extension for guessing the content type, or they know the expected type already. In any case, the default was meant for non-file parts. It actually mentions in the next sentence that for files, a content type should be sent, and if a more appropriate cannot be found it should be `application/octet-stream`. Which makes a lot more sense.

Reply:
Well, at least there is an agreement that RFC is there to be followed. If a part is detected to be a file part, defaulting to `application/octet-stream` is totally acceptable given that it conform to RFC.